packages/vx-event/src/localPoint.ts

```diff
@@ -0,0 +1,14 @@
+import localPointGeneric from './localPointGeneric';
+import { EventType } from './types';
+import { isElement, isEvent } from './typeGuards';
+
+export default function localPoint(nodeOrEvent: Element | EventType, maybeEvent?: EventType) {
+  if (isElement(nodeOrEvent) && maybeEvent) {
+    return localPointGeneric(nodeOrEvent, maybeEvent);
+  }
+  if (isEvent(nodeOrEvent)) {
+    const node = nodeOrEvent.target as Element;
+    if (node) return localPointGeneric(node, nodeOrEvent);
+  }
+  return undefined;

```

Comment:
@hshoff there are a few places in other packages (e.g., [`drag`](https://github.com/hshoff/vx/blob/master/packages/vx-drag/src/Drag.js#L23)) that expect to always get a value from this, I think we should probably return `{ x: 0, y: 0 }` by default.

Reply:
oh i see, you answered my questions below. hmm, not ideal. 

Maybe we could make a new `NullPoint` class and return that instead? `new NullPoint() => { x: null, y: null }`

then in Drag we can instanceof check the returned value and if it's `NullPoint` and handle the nulls there.